keras_nlp/models/bert/bert_tasks.py

```diff
@@ -110,11 +113,17 @@ def from_config(cls, config):
         return cls(**config)
 
     @classproperty
-    def presets(cls):
+    def backbone_presets(cls):
         return copy.deepcopy(backbone_presets)
 
+    @classproperty
+    def presets(cls):
+        return copy.deepcopy(backbone_presets | classifier_presets)
+
     @classmethod
-    @format_docstring(names=", ".join(backbone_presets))
+    @format_docstring(
+        names=", ".join(bert_presets.backbone_presets | classifier_presets)
```

Comment:
@mattdangerw there is some crazy binding issue where your decorator aliases `backbone_presets` with the `classmethod` but also won't let me use it as a `dict`. Worth a discussion.

Reply:
I would just pull this a constant above the class.

`PRESET_NAMES = ", ".join(bert_presets.backbone_presets.keys() + classifier_presets.keys())`

`@format_docstring(names=PRESET_NAMES)`

Would that work?

Reply:
Seems clean!

Reply:
It turns out that `dict.keys()` does not return a `list` but a `dictview` so the above does not work. Luckily what does work is even simpler:

```python
PRESET_NAMES = ", ".join(list(backbone_presets) + list(classifier_presets))
```